From effb59d27d9ad9c1a98c248663d96fcac4a65b40 Mon Sep 17 00:00:00 2001
From: Daniel Charles <daniel.charles@intel.com>
Date: Mon, 25 Nov 2019 14:46:28 -0800
Subject: [PATCH] Fix build issues with bazel

tensorflow-serving is using an older bazel version producing these
building issues.

Signed-off-by: Daniel Charles <daniel.charles@intel.com>
---
 tensorflow_serving/session_bundle/oss/BUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tensorflow_serving/session_bundle/oss/BUILD b/tensorflow_serving/session_bundle/oss/BUILD
index 5ce47e54..25f7c2c3 100644
--- a/tensorflow_serving/session_bundle/oss/BUILD
+++ b/tensorflow_serving/session_bundle/oss/BUILD
@@ -1,4 +1,4 @@
-load("@org_tensorflow//tensorflow/core:platform/default/build_config.bzl", "tf_proto_library")
+load("@org_tensorflow//tensorflow/core/platform:default/build_config.bzl", "tf_proto_library")
 load("//tensorflow_serving:oss_or_google.bzl", "oss_only_cc_test")
 
 licenses(["notice"])  # Apache 2.0
-- 
2.24.0

